laba.py

```diff
@@ -59,6 +59,8 @@ def update_car(self, id, new_value):
 car8 = Car(8, 'Audi', 45000, CarAttributes('Blue', 'Rain', 'Regular'))
 car9 = Car(9, 'Ford', 30000, CarAttributes('Violet', 'Sport', 'Level-1'))
 car10 = Car(10, 'Toyota', 32000. CarAttributes('Green', 'Show', 'Level-2'))
```

Comment:
you have a syntax error here, take a look at the master branch

Reply:
oh, thx, i'll fix it